src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/sizes/recordcomponentnumber/Example3.java

```diff
@@ -13,12 +13,14 @@
 </module>
 */
 
+//non-compiled with javac: Compilable with Java17
+package com.puppycrawl.tools.checkstyle.checks.sizes.recordcomponentnumber;
+
 // xdoc section -- start
-public record MyRecord1(int x, int y, String str) { // ok, public record definition allowed 10
-   ...
-}
+class Example3 {
+  public record MyRecord1(int x, int y, String str) {} // ok, 3 components
 
-private record MyRecord2(int x, int y, String str, Node node) { // violation
-   ...                                // private record definition allowed 3 components
+  private record MyRecord2(int x, int y, // violation, 4 components
```

Comment:
Please make all examples java to be the same.

Reply:
they need changes in the argument number and access modifier as well for the customizations in their config. Remaining everything I tried to keep same. Do you mean anything else to be kept same?

Reply:
Done